lib/local_subnet_manager.hpp

```diff
@@ -4,16 +4,16 @@
 #ifndef LOCAL_SUBNET_MANAGER
 #define LOCAL_SUBNET_MANAGER
 
-#include "subnet_manager.hpp"
 #include "component_list.hpp"
+#include "subnet_manager.hpp"
 #include <cstdint>
 
 class LocalSubnetManager : public SubnetManager
 {
 public:
-  LocalSubnetManager(std::shared_ptr<SpaCommunicator> c) :SubnetManager(c) {}
+  LocalSubnetManager(std::shared_ptr<SpaCommunicator> c) : SubnetManager(c) {}
   void receiveMessage(std::shared_ptr<SpaMessage> message);
- // receiveMessage(uint8_t *buff, uint32_t bufflen);
-  ComponentList components; 
+  // receiveMessage(uint8_t *buff, uint32_t bufflen);
```

Comment:
I'm really not sure. I think the communicators handle the marshaling/unmarshaling?